zk.js/src/merkleTree/merkleTreeConfig.ts

```diff
@@ -208,7 +235,7 @@ export class MerkleTreeConfig {
       await this.getTransactionMerkleTreeAccountInfo(
         transactionMerkleTreePubkey,
       );
-    return transactionMerkleTreeAccountInfo.newest == 1;
+    return transactionMerkleTreeAccountInfo.newest.toString() == "1";
```

Comment:
```suggestion
    return transactionMerkleTreeAccountInfo.newest.toString() === "1";
```

Reply:
while not sure if type coercion happens in this case it's safer to do strict comparison